include/cub3d.h

```diff
@@ -47,24 +49,95 @@
 
 # ifndef DEBUG
 #  define DEBUG 1
+
+# endif
+# ifndef NOT_XPM
+#  define DEBUG 1
```

Comment:
Surely this was meant to be NOT_XPM not "DEBUG"

Reply:
This is fixed and the blame was on me... sorry........